### DISCUSSION TITLE
Analysing SERPs on Large Sclae

### DISCUSSION THREAD

**Discussion by Rushikesh494** [eliasdabbas/advertools#126] [Q&A] [answered]
How do I get  search results for the top 50 pages by using advertools python code
By using the below code its getting top 10 search results:
import matplotlib.pyplot as plt
import advertools as adv
import pandas as pd
pd.set_option('display.max_columns', None)
cx = '9d8154d8a6d42d0c1'
key = 'GOOGLE_KEY'
queries = ['hotels','schools']
serp_ranks = adv.serp_goog(cx=cx, key=key, q=queries,gl=['om'])# imports data
print(serp_ranks)

 Could you please help me

**Comment by eliasdabbas** [comment, marked as answer]
You can use the `start` parameter multiple times like this: 

```python
serp_ranks = adv.serp_goog(cx=cx, key=key, q=queries,gl=['om'], start=[1, 11, 21, 31, 41])
```
This would run the same queries once for each `start` position, ending up with the top 50 results. 

Note: your key was edited to protect its privacy. Watch out not to publish it anywhere, and it's better to deactivate it and create a new one.

**Reply by Rushikesh494** [reply]
Hi, @eliasdabbas thank you very much, It was a really amazing tool. Is there any way to get top 50 results in 1 search request for a particular keyword? it was really helpful to all the digital marketers if we can pull the top 50 results in 1 search request. Since most SEO marketers would look at the bulk keyword tool, 10 requests for a keyword were taking too many requests. could you please work on it, please?

**Reply by eliasdabbas** [reply]
Answered in #128
